requirements-dev.txt

```diff
@@ -16,7 +16,7 @@ m2r2
 nbsphinx
 pydocstyle
 pytest
-sphinx == 7.1.2
+sphinx >=3.4, <=7.1.2
 sphinx_compas_theme >=0.15.18
```

Comment:
the new theme is available now: `sphinx_compas2_theme`

Reply:
there are two options
* `multisection`
* `sidebaronly`
the docs of the them itself are built as "sidebaronly", so you can copy its `conf.py`

Reply:
will do in #195